### PR TITLE
Add the java.import.maven.offline.enabled property

### PR DESCRIPTION
**Pull request by snjeza**
Fixes https://github.com/snjeza/vscode-java/pull/new/issue-2186
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/2187

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -250,6 +250,12 @@
           "description": "Enable/disable the Maven importer.",
           "scope": "window"
         },
+        "java.import.maven.offline": {
```
Comment (rgrunber) [inline package.json:253]: I would call this `java.import.maven.offline.enabled` so it matches the corresponding property on the Gradle side.

File: README.md [hunk 2]
```diff
@@ -205,6 +205,7 @@ The following settings are supported:
 * `java.configuration.workspaceCacheLimit` : The number of days (if enabled) to keep unused workspace cache data. Beyond this limit, cached workspace data may be removed.
 * `java.import.generatesMetadataFilesAtProjectRoot` : Specify whether the project metadata files(.project, .classpath, .factorypath, .settings/) will be generated at the project root. Defaults to `false`.
 * `java.inlayHints.parameterNames.enabled`: Enable/disable inlay hints for parameter names. Supported values are: `none`(disable parameter name hints), `literals`(Enable parameter name hints only for literal arguments) and `all`(Enable parameter name hints for literal and non-literal arguments). Defaults to `literals`.
+* `java.import.maven.offline` : Enable/disable the Maven offline mode. Defaults to `false`.
```
Comment (rgrunber) [inline README.md:208]: Once Also once https://github.com/redhat-developer/vscode-java/pull/2614/files is merged is merged, I would just add this under the `New in 1.10.0` heading.

**Review by rgrunber — COMMENTED** [review]
Overall, looks fine to me.

File: package.json (hunk 1 above)
Comment (snjeza) [reply package.json]: Fixed

File: README.md (hunk 2 above)
Comment (snjeza) [reply README.md]: Fixed.

**Review by rgrunber — APPROVED** [review]

**Pull request merged**